### ISSUE TITLE
'Command may not exist' for commands that do exist

### ISSUE THREAD

**Issue by jyn514** [chriswalz/bit#20]
(sorry, posted a little early)

```
$ $GOPATH/bin/bit 
> bit pull
Your configuration specifies to merge with the ref 'refs/heads/readme'
from the remote, but no such ref was fetched.
Command may not exist exit status 1
```

**Comment by jyn514** [comment]
Same error for `bit -h`:
```
$GOPATH/bin/bit -h
unknown option: -h
usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           <command> [<args>]
Command may not exist exit status 129
```

**Comment by chriswalz** [comment]
What happens when you run `git pull` in the same directory?

Right now `bit -h` doesn't work but you can use `bit help` see all bit specific commands

**Comment by jyn514** [comment]
Right, `git pull` also gives an error - my bug report is not that bit is giving an error, but that it's giving an _additional_ error on top of `git pull`.

**Comment by chriswalz** [comment]
Ah understood. Thanks

**Comment by chriswalz** [comment]
fixed with 97097dc6c4408dc59b687594aaa72a3528053778
